client/gutenberg/editor/calypsoify-iframe.tsx

```diff
@@ -11,8 +11,8 @@ import { localize, LocalizeProps } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
+import AsyncLoad from 'components/async-load';
 import MediaStore from 'lib/media/store';
```

Comment:
Idea for a followup PR: async load the `MediaStore`, too. I don't immediately see how exactly to do this, but the only reason we need the `MediaStore` here is to register the `on( 'change', updateImageBlocks )` callbacks. As the `change` event can come only from the Media Modal, the whole thing could load and register just-in-time.

Reply:
That makes a lot of sense. We should be able to move that logic away from the `calypsoify-iframe` into the media modal. Happy to follow-up with this in another PR once we ship this one. 👍